src/App.css

```diff
@@ -16,13 +16,116 @@
   -moz-osx-font-smoothing: grayscale;
 }
 
-
 /*  Search bar */
 
+.app-header {
+  background-color: #759dd2;
+  height: 100px;
+  font-family: Raleway;
+  padding: 35px 50px;
+  align-items: center;
+}
+::placeholder {
+  color: white;
+  
+}
+.inputBar {
+  color: white;
+  background-color: #759dd2;
+  font-family: Raleway;
+  font-size: 16px;
+  padding: 10px;
+  border: none;
+  border-bottom:thin solid whitesmoke;
+}
 
+.btn {
+  font-family: Raleway;
+  font-size: 14px;
+  padding: 10px;
+  text-transform: uppercase;
+  background: #587ac2;
+  padding: 10px;
+  margin-left: 20px;
+  border-radius: 5px;
+  display: inline-block;
+  border: none;
+}
 
 /* Current weather */
 
+.app_main {
+  background-color: #9ccdf2;
+  align-items: center;
+}
 
+.main_pic .main_pic_icon {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  margin-top: 10px;
+}
+.main_pic {
+  display: flex;
+  justify-content: center;
+}
+.overcast {
+  font-family: Raleway;
+  font-size: 16px;
+  color: white;
+  justify-content: center;
+  margin-top: 30px;
+}
 
+.temp {
+  font-family: Raleway;
+  font-weight: bold;
+  color: #000030;
+  margin-top: 20px;
+}
 
+.humidity_pressure {
+  font-family: Raleway;
```

Comment:
You shouldn't need to set the font-family on every element. If you set it on the main wrappers (example `app_main`), the children should inherit it.